netutils/iperf/CMakeLists.txt

```diff
@@ -30,7 +30,9 @@ if(CONFIG_NETUTILS_IPERF)
     STACKSIZE
     ${CONFIG_NETUTILS_IPERF_STACKSIZE}
     PRIORITY
-    ${CONFIG_NETUTILS_IPERF_PRIORITY})
+    ${CONFIG_NETUTILS_IPERF_PRIORITY}
+    INCLUDE_DIRECTORIES
+    ${CMAKE_BINARY_DIR}/apps/include)
```

Comment:
should  `${CMAKE_BINARY_DIR}/apps/include` add to the search path by default? @xuxin930

Reply:
Good question, wanted to have minimal changes here.
Will wait for @xuxin930 response - or if I will discover any other default configuration relying on build folder include directory I will make it invariant.

Reply:
Done as xuxin930 suggested.